src/Containers/Microsoft.NET.Build.Containers/Tasks/CreateNewImage.cs

```diff
@@ -240,4 +242,108 @@ public void Dispose()
     {
         _cancellationTokenSource.Dispose();
     }
+
+    internal (string[] entrypoint, string[] cmd) DetermineEntrypointAndCmd(string[]? baseImageEntrypoint)
```

Comment:
we need to replicate this logic in the net472 version of the command - the validations there are only looking at EntryPoint/EntrypointArgs and so are breaking after this PR.

Reply:
Where is the net472 version?

Reply:
It's the sibling CreateNewImageToolTask.cs file next to CreateNewImage.cs - this task invokes the `containerize` exe with CLI variables, but does validation of those variables in its `Execute` method so that users get good MSBuild diagnostics.

The ToolTask and the 'normal' task share the same Interface.fs partial class, but are conditionally compiled for net472 or net8.0, respectively.

Reply:
Got it. Shall I look into making the necessary changes?

Reply:
That would be awesome! I _think_ it would be a matter of

* pushing this entrypoint-detection method down into the .Interface.cs class
* Using it in both CreateNewImage tasks separately
* Emitting the new CLI args for the entrypoint args/command/etc in the ToolTask's 'GenerateCommandLine' method
* A brief look at emitting nice diagnostics from the ToolTask

Reply:
I'll look into it. Thanks for the outline.